src/ncp-spinel/SpinelNCPInstance.cpp

```diff
@@ -43,6 +43,33 @@ using namespace wpantund;
 
 WPANTUND_DEFINE_NCPINSTANCE_PLUGIN(spinel, SpinelNCPInstance);
 
+void
+SpinelNCPInstance::handle_ncp_log(const uint8_t* data_ptr, int data_len)
+{
+    static char linebuffer[NCP_DEBUG_LINE_LENGTH_MAX + 1];
+    static int linepos = 0;
+    while (data_len--) {
```

Comment:
It's because I can get away with mutating the arguments here, whereas I didn't want to mutate the arguments in the for loop above.